1-create-cluster.sh

```diff
@@ -12,7 +12,7 @@ cd "$(dirname "${BASH_SOURCE[0]}")"
 #
 # Create or start the cluster
 #
-minikube start --cpus=2 --memory=8192 --disk-size=50g --driver=hyperkit --profile curity
+minikube start --cpus=4 --memory=16384 --disk-size=50g --driver=hyperkit --profile curity
```

Comment:
Does it mean you need at least 16gb RAM to start this? 😓

Reply:
I suppose I was trying to say "If you are running one of the bigger installed, use more RAM". I will say this in the article instead. Changed back to 8GB.